runner/serializers.py

```diff
@@ -164,6 +174,30 @@ class Meta:
         fields = ('id', 'name', 'status', 'tags', 'app', 'inputs', 'outputs', 'status_url', 'created_date', 'job_statuses','execution_id','output_metadata','output_directory','operator_run','job_group','notify_for_outputs','finished_date','message')
 
 
+class RunSerializerCWLInput(RunSerializerPartial):
+
+    inputs = serializers.SerializerMethodField()
+
+    def get_inputs(self, obj):
+        input_data = PortSerializer(obj.port_set.filter(port_type=PortType.INPUT).all(), many=True).data
+        return format_port_data(input_data)
```

Comment:
How should the File object look like? Do we need path/location/size etc, and should it be runnable by Voyager or locally?

Reply:
As previously discussed, we will create a uniform format for inputs and outputs